// roblo/index.js
var Database = require('better-sqlite3');
var db = new Database(__dirname+'/db/rob.db');

// === usr ===
// ins
exports.inUsr=function(name,mail,pss,sid){
var stm=db.prepare('insert into usr values(?,?,?,?)')
try{var row=stm.run(name,mail,pss,sid)}
catch(err){console.log(err);}
};

//upd
exports.upSid=function(sid){
var stm=db.prepare('update usr set sid=? where mail=? )')
try{stm.run(sid)
}catch(err){console.log(err);}
};

exports.selUsr=function(mail,pss){
var stm=db.prepare('select * from usr where mail=? and pss=?');
try{var row=stm.all(mail.pss);
}catch(err){console.log(err);}
return row;
};
// === pre ===
exports.selPre=function(arg2){
var stm=db.prepare('select * from pre where type=?');
try{var row=stm.all(arg2);
}catch(err){console.log(err);}
return row;
};

exports.allPre=function(){
var stm=db.prepare('select * from pre' )
try{var row=stm.all();
}catch(err){console.log(err);}
return row;
};
// ins
exports.inPre=function(name,mail,pss,item){
var stm=db.prepare('insert into pre values(?,?,?,?)')
try{var row=stm.run(name,mail,pss,item)}
catch(err){console.log(err);}
};

//upd
exports.upTyp=function(type,tit){
var stm=db.prepare('update rob set type=? where tit=? )')
try{stm.run(type,tit)
}catch(err){console.log(err);}
};

exports.upTer=function(typ,term){
var stm=db.prepare('update rob set term=? where tit=? )')
try{stm.run(typ,tit)
}catch(err){console.log(err);}
};
